infra/src/test/kotlin/fi/fta/geoviite/infra/ui/pagemodel/inframodel/InfraModelElement.kt

```diff
@@ -103,7 +103,7 @@ internal class E2EFormGroupField(by: By) : E2EViewFragment(by) {
         toggleEdit()
     }
 
-    fun selectNewValue(newValues: List<String>) = apply {
+    fun selectNewValue(newValues: List<String>, toastId: String) = apply {
```

Comment:
Hyvä muutos, paljon fiksumpaa että odotetaan täällä 👍 Jäin tosin miettimään tämän funktion nimeämistä. Puhdas `select` on mielestäni vähän huono verbi kuvaamaan kokonaan uuden arvon luontia (vaikka ymmärrän miksi se sillä nimellä on ollut, kyllähän sieltä dropdownin näkökulmasta uuden luontioptio valitaan.) Joku tyyliin `createNew...` tai `selectCreateNew...` (tai sitten joku variantti `insert`:illä `create`:n sijasta) voisi olla parempi verbi/verbiyhdistelmä tuohon